### PR TITLE
2023-1.1: update mxtools, nyxtools, csxtools, nslsii

### PR DESCRIPTION
**Pull request by JunAishima**

### CODE REVIEW

File: envs/env-py310.yml [hunk 1]
```diff
@@ -75,7 +75,7 @@ dependencies:
   - nslsii >=0.8.0
   - numexpr
   - numpy >=1.20
-  - nyxtools >=0.0.10
+  - nyxtools >=0.0.11
```
Comment (mrakitin) [inline envs/env-py310.yml:78]: Could you please update the name `2023-1.0-py310-tiled` on top of this file too?

File: envs/env-py39.yml [hunk 2]
```diff
@@ -65,7 +65,7 @@ dependencies:
   - memory_profiler
   - mendeleev
   - modestimage
-  - mxtools >=0.0.8
+  - mxtools >=1.0.1
```
Comment (mrakitin) [inline envs/env-py39.yml:68]: I think the env name on top of this file will need to be updated too.

File: envs/env-py39.yml [hunk 3]
```diff
@@ -75,7 +75,7 @@ dependencies:
   - nslsii >=0.8.0
```
Comment (mrakitin) [inline envs/env-py39.yml:75]: Could we also bump the `nslsii` version to be [0.9.0](https://github.com/NSLS-II/nslsii/releases/tag/v0.9.0)? (this applies to the py310 file too.)

**Review by mrakitin — COMMENTED** [review]
This needs minor corrections but looks good overall.

File: envs/env-py310.yml (hunk 1 above)
Comment (JunAishima) [reply envs/env-py310.yml:78]: done in https://github.com/nsls2-conda-envs/nsls2-collection-tiled/pull/11/commits/202e38e7b0e8034e2585d3b582776f43aca9872b

File: envs/env-py39.yml (hunk 2 above)
Comment (JunAishima) [reply envs/env-py39.yml:68]: done in https://github.com/nsls2-conda-envs/nsls2-collection-tiled/pull/11/commits/202e38e7b0e8034e2585d3b582776f43aca9872b

File: envs/env-py39.yml (hunk 3 above)
Comment (JunAishima) [reply envs/env-py39.yml]: done in https://github.com/nsls2-conda-envs/nsls2-collection-tiled/pull/11/commits/202e38e7b0e8034e2585d3b582776f43aca9872b

**Review by mrakitin — APPROVED** [review]

**Comment by mrakitin** [comment]
Deposited to Zenodo at https://zenodo.org/record/7603440.

**Pull request merged**